test/integration/hooks/associations.test.js

```diff
@@ -62,14 +63,14 @@ describe(Support.getTestDialectTeaser('Hooks'), function() {
           , beforeHook = false
           , afterHook = false;
 
-          this.Tasks.beforeUpdate(function(task, options, fn) {
+          this.Tasks.beforeUpdate(function(task, options) {
             beforeHook = true;
-            fn();
+            return Promise.resolve();
```

Comment:
Oh lol, damn.

Yes without Promise.resolve it works aswell, but using `Promise.resolve()` tests the intended async nature.